salt/states/cpan_state.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
```

Comment:
How about using the filename `cpan`? This would be more inline with other state modules, and make more obvious the state declaration of `cpan.installed:`

Reply:
I based it of of the pip_state.py module which happens to be the only one that has `_state` in the name, It's a good suggestion and I"ll make the change :+1:

Reply:
Fixed in https://github.com/saltstack/salt/pull/52395/commits/d8a2a0efc3089d59e54adeb35f79978cd822d5b3